README.md

```diff
@@ -55,46 +49,83 @@ export default function App() {
 
 ### Navigation
 
-If you use react-router, or any kind of client-side routing, you may want to override
-our standard navigation behavior. Navigation happens when a user chooses a use case
-they want to try out. By default, TestBox will use `window.location` to push the iFrame
-to a new URL. You might want to do something more sophisticated.
+Navigation happens when a user chooses a use case they want to try out.
+By default, TestBox will use `window.location` to push the iFrame to a new URL.
 
-There are two ways to interact with the browser SDK for events:
+If you use react-router, or any kind of client-side routing, you may want to override
+our standard navigation behavior. To do so, especify your custom handler on the
+configuration object for `startTestBox` method:
 
 ```javascript
-// Option 1: you can use the config object
-startTestBox({
-    navigateHandler: (url) => {
-        history.push(url);
-    }
-});
+const testboxConfig = {
+    navigateHandler: (url) => { history.push(url) },
+}
 
-// Option 2: you can set the handler directly
-import testbox from "@testboxlab/browser";
+startTestBox(testboxConfig);
 
-testbox.navigateHandler = (url) => {
-    history.push(url);
-};
 ```
 
 ### Auto-login
 
 If you have opted to use our client-side auto-login functionality, you have a bit
-more work to do.
+more work to do. You should create a login handler, which is responsible for both
+logging in to your platform and redirecting the user accordingly.
```

Comment:
If I’m not mistaking the loginHandler is not responsible for redirecting the user, it just returns the redirect url and then the navigateHandler will do the redirect, maybe it's worth to specifying that?

Reply:
Sure! I'll specify that.